src/components/bottomButton/BottomButton.styles.ts

```diff
@@ -10,8 +11,10 @@ export const BottomButtonContainer = styled.div`
   right: 0;
   margin: 0 auto;
   max-width: 412px;
-  background: ${theme.colors.gray01};
-  border-top: 2px solid ${theme.colors.gray02};
+  background: ${({ background }) => background && theme.colors.gray01};
+  border-top: ${({ background }) =>
+    background && `2px solid ${theme.colors.gray02}`};
+
```

Comment:
이런 식으로 한 번에 써 주는 게 더 간결할 것 같아요

```suggestion

  ${({ background }) =>
    background &&
    css`
      background: ${theme.colors.gray01};
      border-top: 2px solid ${theme.colors.gray02};
    `};

```